api/database/seeders/CategorySeeder.php

```diff
@@ -14,15 +14,15 @@ class CategorySeeder extends Seeder
     public function run(): void
     {
         DB::table('categories')->upsert([
-            ['label' => 'Charger'],
-            ['label' => 'Battery'],
-            ['label' => 'Keyboard'],
-            ['label' => 'Motherboard'],
-            ['label' => 'Flex'],
-            ['label' => 'LCD'],
-            ['label' => 'Case'],
-            ['label' => 'Speaker'],
-            ['label' => 'Fan'],
-        ],  ['label'], ['created_at', 'updated_at']);
+            ['id' => 1, 'label' => 'Charger'],
+            ['id' => 2, 'label' => 'Battery'],
+            ['id' => 3, 'label' => 'Keyboard'],
+            ['id' => 4, 'label' => 'Motherboard'],
+            ['id' => 5, 'label' => 'Flex'],
+            ['id' => 6, 'label' => 'LCD'],
+            ['id' => 7, 'label' => 'Case'],
+            ['id' => 8, 'label' => 'Speaker'],
+            ['id' => 9, 'label' => 'Fan'],
+        ], ['id'], ['label', 'created_at', 'updated_at']);
```

Comment:
Should be label, make the label unique so it will work.
Also, id is not good to reference because if have data already. Then the label will change it will cause data discrepancy.

```suggestion
        ], ['label'], ['label', 'created_at', 'updated_at']);
```